.github/workflows/eleventy_build_pr.yml

```diff
@@ -0,0 +1,70 @@
+name: Eleventy Build PR Site
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - ready_for_review
+      - reopened
+jobs:
+  build_deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: n1hility/cancel-previous-runs@v2
+        with: 
+          token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@master
+      - name: Use Node.js 14.17.0
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.17.0
+      - name: Get branch name (merge)
+        if: github.event_name != 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/})" >> $GITHUB_ENV
+      - name: Get branch name (pull request)
+        if: github.event_name == 'pull_request'
+        shell: bash
+        run: echo "BRANCH_NAME=$(echo ${GITHUB_HEAD_REF})" >> $GITHUB_ENV
+      - name: Escape branch name for URL
+        shell: bash
+        run: echo "URLSAFE_BRANCH_NAME=$(echo ${BRANCH_NAME} | sed 's|[^A-Za-z0-9-]|-|g' | sed -E 's|-*([A-Za-z0-9]*.*[A-Za-z0-9]+)-*|\1|')" >> $GITHUB_ENV
+      - name: Report escaped branch name
+        shell: bash
+        run: echo ${URLSAFE_BRANCH_NAME}
+      - name: Build 11ty
+        run: |
+          mkdir dist
+          npm ci --production
+          DOMAIN=${URLSAFE_BRANCH_NAME}.pr.covid19.ca.gov npm run build
+      - name: Write robots.txt
+        run: |
+          echo 'User-agent: *' > docs/robots.txt
+          echo 'Disallow: /' >> docs/robots.txt
```

Comment:
I'm not sure what this robots.txt job will do here, since Covid already writes a default `robots.txt` from the `rootcopy` folder. Something to watch.

Reply:
Oh yeah, we should nuke that bit...